coffeecard/CoffeeCard.Models/DataTransferObjects/User/EmailInUseDTO.cs

```diff
@@ -0,0 +1,10 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace CoffeeCard.Models.DataTransferObjects.User
+{
+    public class EmailInUseDTO
```

Comment:
1) Shold be added to the v2 subfolder
2) In Api V2, we tend to not the the DTO suffix but instead give a more direct name, e.g. `EmailInUseResponse`